jqm-all/jqm-ws/src/main/java/com/enioka/jqm/ws/api/ServiceClient.java

```diff
@@ -65,10 +73,23 @@ public class ServiceClient
 {
     static Logger log = LoggerFactory.getLogger(ServiceClient.class);
 
+    private boolean standaloneMode;
+    private String localIp;
+
     @Activate
     public void onServiceActivation(Map<String, Object> properties)
     {
         log.info("\tStarting ServiceClient");
+        standaloneMode = Boolean.parseBoolean(
+            GlobalParameter.getParameter(DbManager.getDb().getConn(), "wsStandaloneMode", "false"));
+
+        if (standaloneMode) {
```

Comment:
may be add a check if standalone and db != from hsqldb than error "not supported"